Dockerfile.ubuntu

```diff
@@ -0,0 +1,85 @@
+FROM golang:1.13 AS go-builder
```

Comment:
```suggestion
FROM golang:1.13.1 AS go-builder
```

Reply:
@marefr How about we use the latest version, 1.13.4?

Reply:
Grafana builds against 1.13.1 so don't think we want to change this until Grafana build is upgraded to go 1.13.4

Reply:
OK